src/__tests__/patients/search/ViewPatients.test.tsx

```diff
@@ -33,8 +34,8 @@ describe('Patients', () => {
   })
 
   it('should render the search patients component', () => {
-    const wrapper = setup()
+    render()
```

Comment:
There's too many gaps to post a suggested change, but again, I would inline this into the test.

Reply:
I prefer to keep it this way if it's okay. This component seems like there is scope for changes. Removed the extra assignment to render function though.